tools/compogen/pkg/gen/resources/templates/readme.mdx.tmpl

```diff
@@ -7,146 +7,157 @@ description: "Learn about how to set up a VDP {{ .Title }} component https://git
 
 The {{ .Title }} component is {{ .ComponentType.IndefiniteArticle }} {{ .ComponentType }} component that allows users to {{ firstToLower .Description }}.
 It can carry out the following tasks:
-{{ range .Tasks }}
-- [{{ .Title }}](#{{ asAnchor .Title}}){{ end }}
+{{- range .Tasks }}
```

Comment:
[TIL](https://developer.hashicorp.com/nomad/tutorials/templates/go-template-syntax#trim-markers) 🥇 This is gold, makes everything much more readable